### ISSUE TITLE
[Content styleguide] Content doesn’t align with style guidance in VA.gov content style guide (05.08.1)

### ISSUE THREAD

**Issue by RLHecht** [department-of-veterans-affairs/va.gov-team#34953]
#### VFS team name

Public Websites

#### VFS product name

Outreach and Events MVP

#### Point of Contact/Reviewers (Practice area)

Randi Hecht (Content)

---

### Platform Issue

Content doesn't align with style guidance in VA.gov content style guide.

### Issue Details

All times should be in the format ET, MT, PT (Not EST, MST, PST) (Not sure if this is in the scope of this work but pointing it out)
Month abbreviations should have a period at the end.

### Link, screenshot or steps to recreate

<img width="664" alt="Screen Shot 2022-01-07 at 11 52 38 AM" src="https://user-images.githubusercontent.com/48728214/148578094-6221c853-2060-4907-bd3a-54e2aa98208f.png">

### VA.gov Experience Standard

[Category 05, Issue 08.](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References

WCAG SC 3.2.4 AA

### Platform Recommendation

Change styling to ET, MT, or PT
Add period to end of month abbreviation (Jan.)

### VFS Team Tasks to Complete

- [x] Comment on the ticket if there are questions or concerns
- [ ] Close the ticket when the issue has been resolved or validated by your Product Owner. If a team has additional questions or needs Platform help validating the issue, please comment in the ticket. 

## Platform directions
- Update "Issue Title"
- Link to collab cycle touchpoint meeting epic
- Add to the Feedback pipeline on the Platform Product Support board
- Add your feedback
- Add assignees: VFS PM
- Add labels:
  - VFS team label
  - VA.gov Experience Standard Issue number (CCIssueXX.YY) where XX is the category number and YY is the issue number, per the VA.gov Experience Standards
  - Practice area labels (508/accessibility, IA, content, design) per the VA.gov Experience Standards
  - 508 defect label, if applicable (508-defect-0, 508-defect-1, 508-defect-2, 508-defect-3, 508-defect-4)
  - Launch-blocking label, if applicable

**Comment by callen2563** [comment]
@brianalloyd I checked the [designs](https://sketch.com/s/8ec18b80-a905-47fb-a217-5720450befa1) the content formatting is correct these changes listed here need to be updated by dev in staging.

<img width="827" alt="Screen Shot 2022-01-14 at 1 11 17 PM" src="https://user-images.githubusercontent.com/68560244/149564799-fbd7ff40-4cb1-417a-b38e-77004757b9fd.png">

**Comment by srsuddath** [comment]
This ticket has been accomplished by the following PRs
https://github.com/department-of-veterans-affairs/content-build/pull/958
https://github.com/department-of-veterans-affairs/vets-website/pull/20128
https://github.com/department-of-veterans-affairs/content-build/pull/956
